buildspec.yml

```diff
@@ -0,0 +1,19 @@
+version: 0.2
+
+env:
+  variables:
+    GO111MODULE: "on"
+
+phases:
+  pre_build:
+    commands:
+      - go mod download
+  build:
+    commands:
+      - SKIP_INTEG_TEST=1 make test
```

Comment:
very nit: can just make `SKIP_INTEG_TEST` a variable in the `env:` section above

Reply:
I think I prefer having the environment variable defined here just so you know what command it is tied to. But I went ahead and changed it anywhoo.

Reply:
Sure, I don't have a strong feeling either way, my main question was the other I posted about more context on the purpose of this change, I just added this nit comment along the way :-)